add randomness to the credential_offer_uri (#336)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

> This ability to pass the Credential Offer by reference is particularly useful for large requests.

Author: ```suggestion
This ability to pass the Credential Offer by reference is particularly useful for large Credential Offer objects.
```
It's Credential Offer objects

> When the Credential Offer is passed by reference, the Credential Offer can be displayed by the Credential Issuer as a QR code. Below is a non-normative example:

Reviewer: One could argue that the example is not a QR code and that this QR codes may also be used without passing by reference

Author: ```suggestion
The Credential Offer can be displayed by the Credential Issuer as a QR code. Below is a non-normative example of a content of such a QR code when the Credential Offer is passed by reference:
```

Reviewer 2: ```suggestion
When the Credential Offer is displayed as a QR code, it would usually contain the Credential Offer by reference due to the size limitations of the QR codes. Below is a non-normative example:
```

Anyone can render the QR code

```
openid-credential-offer://?

>   credential_offer_uri=https%3A%2F%2Fserver%2Eexample%2Ecom%2Fcredential-offer.json
>   %2FGkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM

Reviewer 3: ```suggestion
  credential_offer_uri=https%3A%2F%2Fserver%2Eexample%2Ecom%2Fcredential-offer
  %2FGkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM
```
I would remove the .json if we want to show a random uri:
`https://server.example.com/credential-offer/GkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM` is a more natural URI compared to 
`https://server.example.com/credential-offer.json/GkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM` imho

Author: i felt the same, but checked RAR examples and they had .jwt examples, so I kept it.
